pkg/option/config.go

```diff
@@ -1448,11 +1448,12 @@ type DaemonConfig struct {
 
 	// Masquerade specifies whether or not to masquerade packets from endpoints
 	// leaving the host.
-	EnableIPv4Masquerade   bool
-	EnableIPv6Masquerade   bool
-	EnableBPFMasquerade    bool
-	EnableBPFClockProbe    bool
-	EnableIPMasqAgent      bool
+	EnableIPv4Masquerade bool
+	EnableIPv6Masquerade bool
+	EnableBPFMasquerade  bool
+	EnableBPFClockProbe  bool
+	EnableIPMasqAgent    bool
+	// expired DNS lookups with still-active connections
```

Comment:
This change still needs to be reverted.

Reply:
I actually reverted this once in one of my previous commits. But I remember the tests were giving me lint issues with that.

Reply:
Can you try again and share the lint issues? Maybe those were on master and it was fixed since then.

Reply:
@pchaigno I just added a test commit to check if it fails

Reply:
It is still failing. I suggest running `contrib/scripts/check-fmt.sh` locally and adding `-w` to the `gofmt` command to write the result back to the original file. If that is still not working, your Golang version might be out of date and you should consider updating to the latest.

Reply:
Hi @christarazi. Actually the test commit is intentional and its early linted in earlier commits. I had a conversation with @pchaigno and he wanted to confirm whether those issues are on my master or my commits.